dev-libs/libconfig/libconfig-1.5.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="A simple C/C++ Configuration File Library"
+DESCRIPTION="Libconfig is a simple library for processing structured \
+configuration files. Libconfig is very compact, a fraction of the size \
+of the expat XML parser library. This makes it well-suited for \
+memory constrained systems like handheld devices."
+HOMEPAGE="http://www.hyperrealm.com/libconfig/"
+COPYRIGHT="2005-2015 Mark A Lindner"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="http://www.hyperrealm.com/libconfig/libconfig-$portVersion.tar.gz"
+CHECKSUM_SHA256="e31daa390d8e4461c8830512fe2e13ba1a3d6a02a2305a02429eec61e68703f6"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libconfig$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libconfig$secondaryArchSuffix = 1.0.0 compat >= 1
+	lib:libconfig++$secondaryArchSuffix = 1.0.0 compat >= 1
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libconfig${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:libconfig$secondaryArchSuffix = 1.0.0 compat >= 1
+	devel:libconfig++$secondaryArchSuffix = 1.0.0 compat >= 1
+	"
+REQUIRES_devel="
+	libconfig$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:bison
+	cmd:flex
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool files
+	rm $libDir/lib*.la
+
+	prepareInstalledDevelLibs libconfig libconfig++
+	fixPkgconfig
+
+	# devel package
+	packageEntries devel \
+		$developDir \
+
```

Comment:
@return: If/when you remove this blank line, don't forget to **_also_** remove the backslash after **`$developDir`** in the previous line, as well as the space before the backslash, of course. Thanks, and sorry if that was obvious ;-)